README.md

```diff
@@ -21,7 +22,8 @@ RubiX has two components: a BookKeeper server and a FileSystem implementation th
 List of things to be done to use RubiX are:
 
 1. Start the BookKeeper server. It can be started via `hadoop jar` command, e.g.:   
->	hadoop jar rubix-bookkeeper-1.0.jar com.qubole.rubix.bookkeeper.BookKeeperServer
+>	hadoop jar rubix-bookkeeper-0.2.11.jar com.qubole.rubix.bookkeeper.BookKeeperServer  
```

Comment:
Can you replace 0.2.11 with <version> ? Then we dont have to change this line every release.